css/acquia_lift.navbar.css

```diff
@@ -96,68 +109,81 @@
   background-color: gray;
 }
 
-/* Nested menu padding */
-#navbar-administration .navbar-tray-horizontal .menu ul li {
-  padding-bottom: .3em;
-}
-
 /* Campaign drop-down */
-#navbar-administration .acquia-lift-menu-item .acquia-lift-campaign {
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item {
+  margin-top: .7em;
+  margin-bottom: .7em;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-campaign,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-preview-option {
+  display: block;
+  width: 100%;
   float: left;
 }
-#navbar-administration .acquia-lift-menu-item .acquia-lift-campaign-edit {
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-campaign-edit,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-delete,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-rename {
+  position: relative;
+  display: block;
   float: right;
-  margin-left: 1.667em;
+  text-align: right;
 }
-
```

Comment:
This removed stuff should still be there.  Needs to be added to scss along with rest of changes from https://github.com/acquia/acquia_lift/commit/b664b81b03be1ce2b8286e1936901f432425b154#diff-1
https://github.com/acquia/acquia_lift/commit/b2f53a9b63f091d4309b851cc1badb9ff7a561f2#diff-1
https://github.com/acquia/acquia_lift/commit/94e7009061911edf291795adf017b61892a7211f#diff-1